packages/components/src/ItemList.test.tsx

```diff
@@ -248,35 +220,56 @@ describe('mouse', () => {
       onSelectionChange.mockClear();
       onSelect.mockClear();
 
-      rightClickItem(itemList, 5);
+      rightClickItem(5);
 
       expect(onSelect).not.toHaveBeenCalled();
       expect(onSelectionChange).not.toHaveBeenCalled();
-
-      itemList.unmount();
     });
   });
 });
 
+function checkFocus(elementList, index) {
+  for (let i = 0; i < 20; i += 1) {
```

Comment:
Shouldn't hardcore `20` here, just use `elementList.length`:
```suggestion
  for (let i = 0; i < elementList.length; i += 1) {
```